cli/src/main.rs

```diff
@@ -106,6 +106,10 @@ enum Cli {
 
         /// The path to the file to be downloaded to the flash
         path: String,
+
+        /// Whether to erase before downloading
+        #[structopt(short, long)]
+        erase: bool,
```

Comment:
Perhaps this should be `--chip-erase` so that it's more obvious that it may be destructive? Just `erase` could be interpreted as "erase the pages I'm writing to".

Reply:
A short if any for chip-erase?

Reply:
I'd say no short? to be on the safe side?